Documentation/configuration.md

```diff
@@ -13,6 +13,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 ##### -name
 + Human-readable name for this member.
 + default: "default"
++ This is the key used in the `-initial-cluster` flag (Ex: `default=http://localhost:2380`). This needs to match the key used the flag if your using static boostrapping.
```

Comment:
... in the `-initial-cluster` flag to recognize itself.

My concern is that users may confuse about it and use this as all keys in the initial-cluster flag.

s/your/you are